autopush/db.py

```diff
@@ -36,34 +39,38 @@ def create_storage_table(tablename="storage", read_throughput=5,
                         )
 
 
-def router_table(tablename="router"):
-    return Table(tablename)
-
-
-def storage_table(tablename="storage"):
-    return Table(tablename)
-
-
 def get_router_table(tablename="router", read_throughput=5,
                      write_throughput=5):
+    """Get the main router table object
```

Comment:
Updating for consistent styling. First line in a multi-line doc string is considered a 'header' and has no period, all other lines must have a period at the end of a sentence.